panel/src/components/Writer/Writer.vue

```diff
@@ -58,6 +65,11 @@ import Toolbar from "./Extensions/Toolbar.js";
 import ToolbarComponent from "./Toolbar.vue";
 
 export const props = {
+  components: {
+    "k-writer-link-dialog": LinkDialog,
+    "k-writer-email-dialog": EmailDialog,
+    "k-writer-toolbar": ToolbarComponent,
+  },
```

Comment:
@distantnative no longer needed in the `develop` branch?

Reply:
Why do you think they are no longer needed?

Reply:
A secondary `export` came in the develop branch, I moved it to the wrong place while rebase, I fixed it.